docs/src/data/meta.ts

```diff
@@ -36,6 +38,16 @@ export const META_INFO: MetaInfo = {
     },
     slug: '[platform]',
   },
+  '/[platform]/studio/create-form': {
+    frontmatter: {
+      metaTitle: 'Amplify UI - Build UI fast with Amplify',
+      metaDescription:
+        'Amplify UI simplifies building accessible, performant, and beautiful applications with cloud-connected capabilities, building blocks, theming, and utilities.',
+      supportedFrameworks: 'all',
+      slug: '[platform]',
```

Comment:
does the `slug` need to be updated here too?

Reply:
Unsure if the slug would need to change to as this works as-is.